ui/src/state/chat/writs.ts

```diff
@@ -165,7 +165,7 @@ export function writsReducer(whom: string) {
         {
           oldest: time,
           newest: time,
-          loadedNewest: false,
```

Comment:
@arthyn could you confirm that this won't cause issues elsewhere?

My thinking is that if this represents a single, newly created message, the window that corresponds to it should assume that it has the newest message, but not older ones.

Reply:
I think true here is right